listenbrainz/webserver/views/stats_api.py

```diff
@@ -754,7 +756,8 @@ def _get_country_code_from_mbids(artist_mbids: set) -> list:
     except requests.RequestException as err:
         current_app.logger.error("Error while getting artist_country_codes, {}".format(err), exc_info=True)
         error_msg = ("An error occurred while calculating artist_map data, "
-                     "try setting 'force_recalculate' to 'false' to get a cached copy if available")
+                     "try setting 'force_recalculate' to 'false' to get a cached copy if available"
+                     "Payload: {}".format(request_data))
```

Comment:
I'd like to log what the response from the endpoint was too, if it's >=400, could we log the text?